tools/code-generation/generator/src/main/resources/com/amazonaws/util/awsclientgenerator/velocity/cpp/AbstractServiceRequestHeader.vm

```diff
@@ -39,6 +39,9 @@ namespace ${serviceNamespace}
         headers.emplace(Aws::Http::HeaderValuePair(Aws::Http::CONTENT_TYPE_HEADER, ${CppViewHelper.computeRequestContentType($metadata)} ));
 #if($metadata.acceptHeader)
         headers.emplace(Aws::Http::HeaderValuePair(Aws::Http::ACCEPT_HEADER, "${metadata.acceptHeader}"));
+#end
+#if ($metadata.awsQueryCompatible)
+        headers.emplace(Aws::Http::HeaderValuePair(Aws::Http::X_AMZN_QUERY_MODE,"True"));
```

Comment:
the specification uses lowercase "true" here.
while http headers must be case-insensitive, I'd prefer to be consistent "just in case".